targeting_terminal.py

```diff
@@ -0,0 +1,38 @@
+import pygame
+
+pygame.font.init()
+
+BLACK = 0, 0, 0
+BLUE = 0, 0, 255
+WHITE = 255, 255, 255
+EVENT_KEY_a = 97
+EVENT_KEY_z = 122
+EVENT_KEY_ENTER = 13
+EVENT_KEY_BACKSPACE = 8
+
+class TargetingTerminal():
+      
+    def __init__(self, DRAWING_SCALE):
+        self.current_text_ = ""
+        self.ui_font_ = pygame.font.SysFont("monospace", 10*DRAWING_SCALE)
+        self.DRAWING_SCALE_ = DRAWING_SCALE
+
+    def update(self , events):
+        for event in events:
+            textWidth = self.ui_font_.size(self.current_text_)[0]
```

Comment:
Optimization: Place this after the ` if event.type == pygame.KEYDOWN:`
Since this textWidth value would only be evaluated if there was a `pygame.KEYDOWN` event, calculate it only when that happens